### PR TITLE
Describe the update and delete feature for the Scheduling Design Changes

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) describes the update and delete feature for the Scheduling Design Changes

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/design/schedule.html

### CODE REVIEW

File: src/design/schedule.md [hunk 1]
```diff
@@ -23,3 +23,24 @@ _Scheduled design changes_
...
 
 1. When complete, click <span class="btn">Save</span>.
+
+## Edit scheduled design change
+
+1. On the _Admin_ sidebar, go to **Content** > _Design_ > **Schedule**.
+
+1. Select the item you want to edit.
+
+1. Make a changes that nesesary.
```
Comment (leslietilling) [inline src/design/schedule.md:33]:
```suggestion
1. Make the necessary changes.
```

File: src/design/schedule.md [hunk 2]
```diff
@@ -23,3 +23,24 @@ _Scheduled design changes_
...
+
+1. When complete, click <span class="btn">Save</span>.
+
+## Delete scheduled design change
+
+1. On the _Admin_ sidebar, go to **Content** > _Design_ > **Schedule**.
+
+1. Select the item you want to delete.
+
+1. Click <span class="btn">Delete</span> on the top panel.
```
Comment (leslietilling) [inline src/design/schedule.md:43]:
```suggestion
1. In the button bar at the top of the page, click <span class="btn">Delete</span>.
```

**Review by leslietilling — COMMENTED** [review]
Just minor edits

**Comment by EvheniiaKolkhiieva** [comment]
Thank you @leslietilling , I have implemented these edits. Please approve.

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**